graph/memstore/quadstore.go

```diff
@@ -143,6 +145,11 @@ type QuadStore struct {
 	index   QuadDirectionIndex
 	horizon int64 // used only to assign ids to tx
 	// vip_index map[string]map[int64]map[string]map[int64]*b.Tree
+
+	valsMu  sync.RWMutex
```

Comment:
Would you mind applying the "mutex hat" as I [mentioned earlier](https://github.com/cayleygraph/cayley/pull/954#discussion_r643769984)?

For example, if to access `vals` you only need `valsMu`, then place then together (with mutex on top), separated from other variables with a line break:
```go
// ...

valsMu sync.RWMutex
vals map[string]int64

// ...
```

or group with anonymous struct:

```go
vals struct{
   sync.RWMutex
   m map[string]int64
}
```

Reply:
I don't it clean for code